package.json

```diff
@@ -19,8 +19,7 @@
     "dr:surge": "node dr-surge.js",
     "build": "webpack --config webpack.prod.js && npm run dr:surge",
     "api:refresh": "./hack/pullapi.sh",
-    "api:generate": "npm run api:generate:ams && npm run api:generate:kas",
```

Comment:
also probably need to update module name to app-services-ui

Reply:
You mean federated module name? I'm not sure about how this will affect things right now

Reply:
I do.